lte/gateway/c/core/oai/tasks/sgw/sgw_handlers.cpp

```diff
@@ -1635,8 +1635,9 @@ status_code_e sgw_handle_nw_initiated_actv_bearer_rsp(
     pgw_base_proc_t* base_proc1 = LIST_FIRST(
         spgw_context->sgw_eps_bearer_context_information.pending_procedures);
     LIST_REMOVE(base_proc1, entries);
-    free_wrapper((void**)&spgw_context->sgw_eps_bearer_context_information
-                     .pending_procedures);
+    delete spgw_context->sgw_eps_bearer_context_information.pending_procedures;
```

Comment:
Can't we use `free_cpp_wrapper` here?

Reply:
It's possible to use free_cpp_wrapper and corrected it now